chia/full_node/block_height_map.py

```diff
@@ -184,6 +184,10 @@ async def _load_blocks_from(self, height: uint32, prev_hash: bytes32):
                             ordered[bytes32.fromhex(r[0])] = (r[2], bytes32.fromhex(r[1]), r[3])
 
             while height > window_end:
+                if prev_hash not in ordered:
+                    raise ValueError(
+                        f"block with header hash {prev_hash.hex()} is missing from your blockchain database"
```

Comment:
The hash is going to be pretty long isn't it?  Even when the values aren't long I like to put the data bit at the end of the message.

```suggestion
                        f"block with header hash is missing from your blockchain database: {prev_hash.hex()}"
```